src/tree/networks/NetworkTreeItem.ts

```diff
@@ -5,20 +5,27 @@
 
 import { Network } from "dockerode";
 import { AzExtParentTreeItem, AzExtTreeItem } from "vscode-azureextensionui";
+import { builtInNetworks } from "../../constants";
 import { ext } from "../../extensionVariables";
 import { getThemedIconPath, IconPath } from '../IconPath';
 import { LocalNetworkInfo } from "./LocalNetworkInfo";
 
 export class NetworkTreeItem extends AzExtTreeItem {
-    public static contextValue: string = 'network';
-    public contextValue: string = NetworkTreeItem.contextValue;
+    public static allContextRegExp: RegExp = /Network$/;
+    public static removeableNetworkRegExp: RegExp = /^removeableNetwork$/i;
+
     private readonly _item: LocalNetworkInfo;
 
     public constructor(parent: AzExtParentTreeItem, itemInfo: LocalNetworkInfo) {
         super(parent);
         this._item = itemInfo;
     }
 
+    public get contextValue(): string {
+        const prefix = builtInNetworks.includes(this._item.networkName) ? 'unremoveable' : 'removeable';
```

Comment:
Should we consider `defaultNetwork` and `customNetwork`, instead of `unremoveableNetwork` and `removeableNetwork`? I'm ok with either, just a thought. @philliphoff as well.